lib/packwerk/checker.rb

```diff
@@ -6,6 +6,13 @@ module Checker
     extend T::Sig
     extend T::Helpers
 
+    class TodoLocation < T::Enum
+      enums do
+        PackageReferencingConstant = new
+        PackageDefiningConstant = new
+      end
+    end
```

Comment:
I like this idea but I feel like we might want to bikeshed on the names a bit. On a higher level this is indicating the package which has committed the offense, we don't even need to consider the "todo" file at all. It's basically answering the question of "who's violation is this?" Maybe something like this:

```ruby
class OffendingPackage < T::Enum
  enums do
    ReferencePackage = new
    DefinitionPackage = new
  end
end
```